identifiers/country-au/aliases.csv

```diff
@@ -1,5 +1,9 @@
 id,sameAs,sameAsNote
```

Comment:
@HKSenior The above suggestions is slightly tweaked. i.e. nsw is changed to WA. In assemblies.csv we do not have an option for putting comments. Also, we are not making changes in alias.csv because this is the case in which North West Central and Moore assemblies has been abolished and amalgamated as Mid West.